pkg/restapi/issuer/operation/operations.go

```diff
@@ -259,8 +262,18 @@ type Operation struct {
 	governanceProvider GovernanceProvider
 	routeSvc           routeService
 	messenger          service.Messenger
+	tlsConfig          *tls.Config
+	oidcClientID       string
+	oidcClientSecret   string
+	oidcCallbackURL    string
```

Comment:
Issuer adapter can support multiple issuers. These need to be part of each issuer profile along with issuer oidc url https://github.com/trustbloc/edge-adapter/pull/411/files#diff-d348a1fc808c8f9c0549391446679e7b7f25d533db5c06e982b16be62c98f331R25.